src/index.html

```diff
@@ -8,6 +8,77 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 main">
+      <div class="stars-star">
+      </div>
```

Comment:
```suggestion
      <div class="stars-star"></div>
```
И по аналогии должно быть везде так